bin/convert_dpc_folder.rb

```diff
@@ -47,7 +50,12 @@
   exit(false)
 end
 
-converter_args = [ options[:source], options[:target], options[:item_id_length], options[:checksums], options[:copy] ]
+converter_args = [ source: options[:source],
```

Comment:
Sorry.  That's a remnant of an incomplete conversion to named arguments for the initializer.  I'll fix.